praise-participants.md

```diff
@@ -0,0 +1,60 @@
+## Title
+Praise Participants
+
+## Patlet
+After an inner source contribution it's important to thank the contributor for their time and effort.
+This pattern gives guidance for a way to do so that not only effectively acknowledges the contribution but also endgenders further engagement from the contributor and others.
+
+## Problem
+How can we properly express our gratitude to a contributor for their inner source contribution to a project?
+It can be easy to forget to do so or not know the words or medium to use for adequate effect and sincerity.
+A pattern in this area makes it easy to do and ensures that the message comes across clearly and sincerely.
+
+## Context
+* You are the trusted committer or maintainer on an inner source project.
+* You value the community of contributors and want to maintain and grow it.
+
+## Forces
+You are busy, which makes it easy to forget some of the soft touches like praise and thanks.
+You may not be someone that is comfortable in social situations or good with words.
+
+## Solutions
+It feels good to anyone to be recognized by others.
+In a professional setting, increased recognition is also an avenue to increased influence and growth.
+Any time someone gives to your inner source project, recognize them with a sincere "thank you".
+
+For non-trivial contributions (all code contributions and also significant time contributions), say thank you via the following mechanisms:
+
+1. Call out the person by name in any chat location (e.g. _Slack_) where you organize your project activity.  Let everyone know what they did and thank them publicly.  Example:
+
+> Everyone @here give a high-five to @andrew.clegg for updating the _rcs-viewer_ to the latest version of the _hebo-client_ (https://github.com/rcs/rcs-viewer/pull/81).
+Thanks for helping keep this library up-to-date, Andy!
+
+2. Send an email to them and their manager (cc'd) privately thanking them for the contribution.
```

Comment:
Or you could @-mention the manager in the merge commit.

Reply:
This works well for code contributions. Something that IMHO should be valued as well: Feedback in code reviews, mentoring, talking colleagues into contributing etc.

Reply:
Thanks for this feedback.  I'm thinking of avoiding making any text changes here because:
* There are probably many ways of reaching out to the manager, but it's not reasonable to list them all.  The merge notification mail is a more universal possibility (e.g. _BitBucket_ doesn't always have a custom merge commit) and allows for more direct wording given its private nature.  The text here is sufficient to give the idea of what we want to happen such that the reader can decide the best way to apply the principle in their own situation (as you've done!)
* An email can be sent for those non-code contributions as well, and the introductory sentence to this section already calls out doing the same thanks for non-trivial time contributions as well.

Reply:
+1 to avoid listing specific ways of reaching out to the manager.

Reply:
👍